azext_iot/central/params.py

```diff
@@ -38,20 +38,10 @@ def load_central_arguments(self, _):
     """
```

Comment:
Would you also be able to fix the typo for me on line 33? 

"scroll = deliver errors as they arrive, json = summarize results as json, csv = summarize results as json", -> "scroll = deliver errors as they arrive, json = summarize results as json, csv = summarize results as csv",